### PR TITLE
Ocr crmops p5

### PR DESCRIPTION
**Pull request by crm-ops**
@fpatel,

Please have a look at these commits. 

Kind Regards

Sébastien

### CODE REVIEW

File: src/test/java/com/parkit/parkingsystem/integration/ParkingDataBaseIT.java [hunk 1]
```diff
@@ -33,33 +43,211 @@ private static void setUp() throws Exception{
...
+
     }
 
+
+
+    // test 30 mins free parking for new client
+            //covered in unit test
+    // test 5% discount for returning client
+            //covered in unit test
+
```
Comment (fpatelm) [inline src/test/java/com/parkit/parkingsystem/integration/ParkingDataBaseIT.java]: You can remove these comments or place them relatively to each test

File: src/test/java/com/parkit/parkingsystem/integration/ParkingDataBaseIT.java [hunk 2]
```diff
@@ -33,33 +43,211 @@ private static void setUp() throws Exception{
...
     }
 
     @Test
-    public void testParkingACar(){
+    public void testParkingACar() throws Exception {
+        when(inputReaderUtil.readSelection()).thenReturn(1);
+        when(inputReaderUtil.readVehicleRegistrationNumber()).thenReturn("ABCDEF");
         ParkingService parkingService = new ParkingService(inputReaderUtil, parkingSpotDAO, ticketDAO);
         parkingService.processIncomingVehicle();
         //TODO: check that a ticket is actualy saved in DB and Parking table is updated with availability
```
Comment (fpatelm) [inline src/test/java/com/parkit/parkingsystem/integration/ParkingDataBaseIT.java]: is this TODO done?

File: src/test/java/com/parkit/parkingsystem/integration/ParkingDataBaseIT.java [hunk 3]
```diff
@@ -33,33 +43,211 @@ private static void setUp() throws Exception{
...
         dataBasePrepareService = new DataBasePrepareService();
+        dataBasePrepareService.clearDataBaseEntries();
     }
 
     @BeforeEach
     private void setUpPerTest() throws Exception {
-        when(inputReaderUtil.readSelection()).thenReturn(1);
-        when(inputReaderUtil.readVehicleRegistrationNumber()).thenReturn("ABCDEF");
-        dataBasePrepareService.clearDataBaseEntries();
+      //  when(inputReaderUtil.readSelection()).thenReturn(1);
```
Comment (fpatelm) [inline src/test/java/com/parkit/parkingsystem/integration/ParkingDataBaseIT.java]: remove not used code

File: src/main/java/com/parkit/parkingsystem/util/InputReaderUtil.java [hunk 4]
```diff
@@ -1,38 +1,54 @@
...
+ */
+
+
 package com.parkit.parkingsystem.util;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
+// svige replace Log4j support by sl4j
+//import org.apache.logging.log4j.LogManager;
+//import org.apache.logging.log4j.Logger;
```
Comment (fpatelm) [inline src/main/java/com/parkit/parkingsystem/util/InputReaderUtil.java]: this can be removed

**Review by fpatelm — CHANGES_REQUESTED** [review]
Minor changes, basically removing commented code and comments.
Good Job!

**Comment by crm-ops** [comment]
@fpatelm 
Thank you for the review comments - all redundant comments removed - test suites run all passing - please review and validate project.

**Pull request merged**